### PR TITLE
chore: update migration summary

### PR DESCRIPTION
**Pull request by blackchoey**

### CODE REVIEW

File: packages/fx-core/templates/core/v3Migration/migrationReport.md [hunk 1]
```diff
@@ -1,9 +1,37 @@
...
+
+If you're using VS Code Teams Toolkit, there're some changes to existing features you need to aware:
+
+### Environment management
+1. All the environment files will be gitignored by default.
+2. When create new environment, you need to fill customized fields in the new `.env.{env_name}` file. Usually you need to provide values for all environment values with `CONFIG__` prefix.
+3. When create new environment, you need to manually create `templates/azure/azure.parameters.{env_name}.json` as Azure provision parameters and fill the parameter values accordingly.
+
+### Launch your app
+1. When launch your app for a remote environment, Teams Toolkit will no longer ask you to select an environment. You need to manually change environment name in `${dev:teamsAppId}` in `.vscode/launch.json` to launch your app for certain environment.
```
Comment (jidddddd) [inline packages/fx-core/templates/core/v3Migration/migrationReport.md:27]: When launching your app for a remote environment, Teams Toolkit will no longer ask you to select an environment. You need to manually change environment name for `${dev:teamsAppId}` in `.vscode/launch.json` to launch your app for a certain environment.

File: packages/fx-core/templates/core/v3Migration/migrationReport.md [hunk 2]
```diff
@@ -1,9 +1,37 @@
...
+
+### Provision SQL databases
+1. When you provision a new environment, you need to provide values for `STATE__FX_RESOURCE_AZURE_SQL__ADMIN` and `SECRET_FX_RESOURCE_AZURE_SQL__ADMINPASSWORD` in `.env.{env_name}` which is required input to create SQL databases.
```
Comment (jidddddd) [inline packages/fx-core/templates/core/v3Migration/migrationReport.md:30]: 1. When you provision a new environment, you need to provide values for `STATE__FX_RESOURCE_AZURE_SQL__ADMIN` and `SECRET_FX_RESOURCE_AZURE_SQL__ADMINPASSWORD` in `.env.{env_name}` which are required inputs for creating SQL databases.

File: packages/fx-core/templates/core/v3Migration/migrationReport.md [hunk 3]
```diff
@@ -1,9 +1,37 @@
...
+    > If you're provision an existing environment, you don't need to take this step.
```
Comment (jidddddd) [inline packages/fx-core/templates/core/v3Migration/migrationReport.md:31]: If you're provisioning an existing environment, you don't need this step.

File: packages/fx-core/templates/core/v3Migration/migrationReport.md [hunk 4]
```diff
@@ -1,9 +1,37 @@
...
+2. You need to grant permission to user assigned identity manually after provision a new environment. This [document](https://aka.ms/teamsfx-add-azure-sql) includes tutorials about how to access SQL databases using user assigned identity.
```
Comment (jidddddd) [inline packages/fx-core/templates/core/v3Migration/migrationReport.md:32]: 2. You need to grant permission to user assigned identity manually after provisioning a new environment.

File: packages/fx-core/templates/core/v3Migration/migrationReport.md [hunk 5]
```diff
@@ -1,9 +1,37 @@
...
+
+### Provision APIM service
+1. When you provision any environment, you need to provide values for `APIM__PUBLISHEREMAIL` and `APIM__PUBLISHERNAME` in `.env.{env_name}` which is required input to create or update APIM services.
```
Comment (jidddddd) [inline packages/fx-core/templates/core/v3Migration/migrationReport.md:35]: 1. When you provision an environment, you need to provide values for `APIM__PUBLISHEREMAIL` and `APIM__PUBLISHERNAME` in `.env.{env_name}` which are required inputs for creating or updating APIM services.

File: packages/fx-core/templates/core/v3Migration/migrationReport.md [hunk 6]
```diff
@@ -1,9 +1,37 @@
...
+2. You need to manually create AAD app for APIM service when provision a new environment. This [document](https://aka.ms/teamsfx-add-azure-apim) includes tutorials about how to create AAD app for APIM service.
```
Comment (jidddddd) [inline packages/fx-core/templates/core/v3Migration/migrationReport.md:36]: 2. You need to manually create an Azure Active Directory app for APIM service when provisioning a new environment. This [document](https://aka.ms/teamsfx-add-azure-apim) includes tutorials about how to create an Azure Active Directory app for APIM service.

**Review by jidddddd — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Pull request merged**